Actions/DetermineArtifactUrl/action.yaml

```diff
@@ -22,6 +22,9 @@ inputs:
     required: false
     default: '{"insiderSasToken":""}'
 outputs:
+  SettingsJson:
```

Comment:
Why would this action output settings?

Reply:
DetermineArtifactUrl updates the settings (in the "old" days it re-wrote the ENV:SETTINGS)
We could instead return the artifactURL, but then we would have to do more changes to CI/CD in order to get this artifact into the settings - or use it from outside the setting.

I didn't want to do major restructuring - instead I was aiming at creating the least disruptive change for removing the ENV problem.